WORKSPACE

```diff
@@ -37,6 +37,20 @@ go_repository(
     version = "v0.0.0-20200203144150-6677ee5c7211",
 )
 
+go_repository(
+    name = "testifiy_assert",
```

Comment:
This is a new third-party dependency that makes it harder to import the code to the internal repository, is it possible to avoid using it here? Just use manual checks with `t.Errorf()` with explicit error messages.

Reply:
Removed assert